<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/images/loginwindow.jpg);</string>
  </property>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>账号(仅能输入字母或数字)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="code">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>密码(仅能输入字母或数字)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
